structured_experiments/2022.02.17.synthetic_generation/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3"
 services:
   train_test_experiment:
     image: test_experiment
-    container_name: test_experiment
+    container_name: test_experiment2
```

Comment:
Moving forward, try to make the `container_name` more informational like `synthetic_experiment` just to help keep organized

No need to change, just moving forward 👍

Reply:
Thank you